### PR TITLE
support any tag that starts with 'fastblock'

### PR DESCRIPTION
**Pull request by mike-parkhill**

### CODE REVIEW

File: .github/workflows/docker_hub_build.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
           tags: docknetwork/dock-substrate:${{ github.ref_name}}
 
       - name: Build release fastblock image
-        if: github.ref_name == 'fastblock'
+        if: startsWith(github.ref_name, 'fastblock')
```
Comment (olegnn) [inline .github/workflows/docker_hub_build.yml:36]: So to publish `fastblock` images for `mainnet`, `testnet` we would need to push 2 more tags like `fastblock_mainnet` and `fastblock_testnet`, right? And one more for the latest version aka `fastblock_master`?

Won't be more convenient and reliable to build `fastblock` for `mainnet` and `testnet` tags by default, and do the same for the latest master commit?

Comment (mike-parkhill) [reply .github/workflows/docker_hub_build.yml:36]: > So to publish fastblock images for mainnet, testnet we would need to push 2 more tags like fastblock_mainnet and fastblock_testnet, right? And one more for the latest version aka fastblock_master

Yeah, that's how it is now

> Won't be more convenient and reliable to build fastblock for mainnet and testnet tags by default, and do the same for the latest master commit?

So, every merge just automatically update all three images? Is that how we normally work? My impression was that we didn't normally update all three at once.

Or, do you mean if you update the mainnet tag also update fastblock-mainnet, similar for testnet. And then for merges to master update the master image and fastblock-master at the same time?

Comment (olegnn) [reply .github/workflows/docker_hub_build.yml:36]: > Or, do you mean if you update the mainnet tag also update fastblock-mainnet, similar for testnet. 

Yes, my proposal is the following sequence of actions:

- `mainnet` tag publishes two images: `mainnet` and `fastblock_mainnet`.
- `testnet` tag publishes two images: `testnet` and `fastblock_testnet`.
- Push to master publishes one image: `fastblock_master`.

Having `fastblock_mainnet`tag  which must be always the same as `mainnet` tag (same for others) looks unnecessary and may lead to inconsistency.

Comment (mike-parkhill) [reply .github/workflows/docker_hub_build.yml]: Made those changes.

**Review by olegnn — APPROVED** [review]

**Pull request merged**